modules/arm_plugin/scripts/arm_cpu_plugin_build.sh

```diff
@@ -145,24 +150,39 @@ ARCHDIR=`ls $OPENVINO_HOME/bin` && \
 cd $DEV_HOME || fail 12 "OpenVINO build failed. Stopping"
 
 #OpenVINO python
+# prepare cross-enviroment
+/usr/bin/python3 -m crossenv /opt/python3.7_arm/bin/python3.7m /tmp/cross_venv
+. /tmp/cross_venv/bin/activate && \
+cross-python -m pip install wheel && \
+cross-python -m pip install numpy==1.20.0
+
+# copy licencing directory
+cp -rf $OPENVINO_HOME/licensing $OPENVINO_HOME/src/bindings/python/
+
 [ "$UPDATE_SOURCES" = "clean" -a -e $OPENVINO_HOME/pbuild ] && rm -rf $OPENVINO_HOME/pbuild
 mkdir -p $OPENVINO_HOME/pbuild && \
 cd $OPENVINO_HOME/pbuild && \
 cmake -DInferenceEngineDeveloperPackage_DIR=$OPENVINO_HOME/build \
-      -DENABLE_PYTHON=ON -DPYTHON_EXECUTABLE="/opt/python3.7_arm/bin/python3.7m" \
+      -DOV_CPACK_RUNTIMEDIR="/arm_cpu_plugin/armcpu_package/runtime/lib/aarch64" \
+      -DENABLE_PYTHON=ON -DPYTHON_EXECUTABLE=`which cross-python` \
+      -DENABLE_WHEEL=ON \
```

Comment:
I'm wondering whether we have proper wheel package out of this? I supposed that `ie_wheel` target is available only with main OpenVINO build. I mean that we likely need to build OV + ARM plugin together using:
```cmake
cmake -DIE_EXTRA_MODULES=<path to arm plugin> -DENABLE_WHEEL=ON -S <path to OpenVINO sources> -B <Builddir>
```

Reply:
Please look here: https://github.com/openvinotoolkit/openvino_contrib/blob/128ea4a093ab3f6761877a33180e6d82bc2d5369/modules/arm_plugin/scripts/arm_cpu_plugin_build.sh#:~:text=%23Build%20OpenVINO,BUILD_JOBS%20%26%26%20%5C . OV is built together with ARM plugin. That code highlighted by you is the part of Python packages compilation.

Reply:
Ok, I see, thanks.

But for this we also need an additional fix https://github.com/openvinotoolkit/openvino/pull/13197 which enables wheel build on top of OpenVINO Developer package. 

Currently, you installs only python modules to the wheel without C++ libraries.